src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetMetaData.java

```diff
@@ -11,4 +12,7 @@ public interface SnowflakeResultSetMetaData {
   int getColumnIndex(String columnName) throws SQLException;
 
   int getInternalColumnType(int column) throws SQLException;
+
+  @SnowflakeJdbcInternalApi
```

Comment:
Should it be internal?

Reply:
Here it couldn't be. It is planned to be public